include/config.hpp

```diff
@@ -39,10 +44,19 @@
 /*
  * Motors config
  */
-#define FRONT_LEFT_ESC_PIN 25
-#define FRONT_RIGHT_ESC_PIN 33
-#define BACK_LEFT_ESC_PIN 27
-#define BACK_RIGHT_ESC_PIN 26
+#if UAV_TYPE == UAV_TYPE_QUADCOPTER
```

Comment:
I believe you can shorten this as follows:

```c++
#define FRONT_LEFT_ESC_PIN 25
#define FRONT_RIGHT_ESC_PIN 33
#define BACK_LEFT_ESC_PIN 27
#define BACK_RIGHT_ESC_PIN 26


#if UAV_TYPE == UAV_TYPE_HEXACOPTER
#define MIDDLE_LEFT_ESC_PIN 23   // temporary pin placement
#define MIDDLE_RIGHT_ESC_PIN 22  // temporary pin placement
#endif
```

Also, remove the indentation in front of all `#define`s